brainfuck/compiler.go

```diff
@@ -0,0 +1,70 @@
+package brainfuck
+
+type Compiler struct {
+	code       string
+	codeLength int
+	position   int
+
+	instructions []*Instruction
+}
+
+func NewCompiler(code string) *Compiler {
+	return &Compiler{
+		code:         code,
+		codeLength:   len(code),
```

Comment:
I think this is redundant here. If you need to know about code length, you can just get it whenever you want. No need to store it in the struct obj.

Reply:
fixed